src/test/regress/sql/alter_table_set_access_method.sql

```diff
@@ -114,6 +113,41 @@ SELECT alter_table_set_access_method('index_table', 'columnar');
 SELECT indexname FROM pg_indexes WHERE schemaname = 'alter_table_set_access_method' AND tablename = 'index_table';
 SELECT a.amname FROM pg_class c, pg_am a where c.relname = 'index_table' AND c.relnamespace = 'alter_table_set_access_method'::regnamespace AND c.relam = a.oid;
 
+CREATE TABLE heap_tbl (
+  c1 CIRCLE,
+  c2 TEXT,
+  i int4[],
+  p point,
+  a int,
+  EXCLUDE USING gist
+    (c1 WITH &&, (c2::circle) WITH &&)
+    WHERE (circle_center(c1) <> '(0,0)'),
+  EXCLUDE USING btree
+    (a WITH =)
+	INCLUDE(p)
+	WHERE (c2 < 'astring')
+);
+
+CREATE INDEX heap_tbl_gin ON heap_tbl USING gin (i);
+CREATE INDEX heap_tbl_gist ON heap_tbl USING gist(p);
+CREATE INDEX heap_tbl_brin ON heap_tbl USING brin (a) WITH (pages_per_range = 1);
+
+CREATE INDEX heap_tbl_hash ON heap_tbl USING hash (c2);
+ALTER TABLE heap_tbl ADD CONSTRAINT heap_tbl_unique UNIQUE (c2);
+
+CREATE UNIQUE INDEX heap_tbl_btree ON heap_tbl USING btree (a);
+ALTER TABLE heap_tbl ADD CONSTRAINT heap_tbl_pkey PRIMARY KEY USING INDEX heap_tbl_btree;
+
+SELECT alter_table_set_access_method('heap_tbl', 'columnar');
+
+SELECT indexname FROM pg_indexes
+WHERE schemaname = 'alter_table_set_access_method' AND
+      tablename = 'heap_tbl'
+ORDER BY tablename;
+
+SELECT conname FROM pg_constraint
+WHERE conrelid = 'heap_tbl'::regclass
+ORDER BY conname;
```

Comment:
Can you put these `SELECT`s before the `alter_table_set_access_method` too so it's easier to see the change?